tests/system/test_arrow.py

```diff
@@ -31,17 +37,35 @@
     ),
 )
 def test_list_rows_nullable_scalars_dtypes(
-    bigquery_client,
-    scalars_table,
-    scalars_extreme_table,
-    max_results,
-    scalars_table_name,
+    bigquery_client: bigquery.Client,
+    scalars_table: str,
+    scalars_extreme_table: str,
+    max_results: Optional[int],
+    scalars_table_name: str,
 ):
     table_id = scalars_table
     if scalars_table_name == "scalars_extreme_table":
         table_id = scalars_extreme_table
+
+    # Avoid INTERVAL columns until they are supported by the BigQuery Storage
+    # API and pyarrow.
```

Comment:
I would prefix this reminder with a "TODO:", more chance of actually finding it later when searching for these kind of things.